src/test/test_integration.py

```diff
@@ -218,15 +214,3 @@ def test_narrative_update_event(self):
             'copied': '1/2/3'
         }
         self.assertEqual(msg_data['doc'], check_against)
-        # Dumb way to wait for the elasticsearch document to save to the index
```

Comment:
We no longer need to test this as this is the responsibility of elasticsearch_writer now.